### ISSUE TITLE
SI issues for POIs (CV wrecks in particular)

### ISSUE THREAD

**Issue by Stan2112** [RavienGaming/Reforged-Eden-2#52]
### Reforged Eden 2 Build Number

4

### Game mode

Survival

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

"Crashed Dive 1" (BA_CrashedCV_Dive1) has SI problems.  I assume since it's a vanilla POI ("BA_..."), this isn't an RE2 issue but I'm starting a report to add all the problematic POIs in one place (maybe?)

**Comment by RavienGaming** [comment]
Yes we cannot fix vanilla POIs. However that particular POI group will need to be replaced with a custom one so I will mark this as "fixed later" as we just need to make the POIs and replace the groups.
If you find any POI that falls apart when you touch it and the blueprint begins with R2 or RE2 when using the di menu please feel free to report them here though as those we can fix.

**Comment by Ishantil** [comment]
Crashed_HM-01 and Crashed_Goblin also have this issue.

**Comment by RavienGaming** [comment]
Duplicate of #39
